maestro-cli/src/main/java/maestro/cli/report/TestDebugReporter.kt

```diff
@@ -125,10 +127,11 @@ object TestDebugReporter {
     fun getDebugOutputPath(): Path {
         if (debugOutputPath != null) return debugOutputPath as Path
 
-        val dateFormat = "yyyy-MM-dd_HHmmss"
-        val dateFormatter = DateTimeFormatter.ofPattern(dateFormat)
-        val folderName = dateFormatter.format(LocalDateTime.now())
-        val debugOutput = Paths.get(debugOutputPathAsString ?: System.getProperty("user.home"), ".maestro", "tests", folderName)
+        val preamble = if(flattenDebugOutput) arrayOf("") else arrayOf(".maestro", "tests")
```

Comment:
@Fishbowler Instead of having this if in every line, let's create a method that returns the `debugOutput` when `flatterDebugOutput` is `true` and another method for when it's `false`